### ISSUE TITLE
Variance explained 

### ISSUE THREAD

**Issue by yuyiyi** [atlab/cov-est#5]
Dear, author, Thanks for sharing the code! I notice that the latent factors generated using the latent+sparse model correlated with the first several SVD components of the covariance matrix to some extent. I would like to know how much of the variance is explained by the latent+sparse model compare to SVD components. Would you be able to provide some thoughts and instructions on doing that?

**Comment by dimitri-yatsenko** [comment]
Hmm.. We did not calculate variance explained.  We used a different validation loss function derived from the Kullback-Leibler Divergence.  The paper discusses at some length why this approach is justified over other loss functions.  We did not compare SVD in the paper (which is equivalent to PCA, not allowing for individual variance) but did compare the Factor Analysis model, which does model variance that's specific to each unit.
